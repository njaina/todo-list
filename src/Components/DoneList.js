   const DoneList = ({ done }) => (
    <>
      {done.map((todo,index) => (
        <div className="response">
          <div className="done-elt" key={todo+index}>
            {todo}
          </div>
        </div>
      ))}
    </>
  );
  export default DoneList;