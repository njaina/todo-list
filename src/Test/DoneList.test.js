import React from "react";
import renderer from "react-test-renderer";
import DoneList from "../Components/DoneList";

describe("DoneList component", () => {
  it("renders correctly", () => {
    const done = ["item 1", "item 2", "item 3"];
    const tree = renderer.create(<DoneList done={done} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
