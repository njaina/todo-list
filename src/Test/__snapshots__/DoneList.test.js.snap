// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoneList component renders correctly 1`] = `
<div>
  <div
    className="response"
  >
    <div
      className="done-elt"
    >
      item 1
    </div>
  </div>
  <div
    className="response"
  >
    <div
      className="done-elt"
    >
      item 2
    </div>
  </div>
  <div
    className="response"
  >
    <div
      className="done-elt"
    >
      item 3
    </div>
  </div>
</div>
`;
